src/Entity/Address.php

```diff
@@ -49,9 +49,20 @@ class Address
      */
     private $city;
 
-    public function __construct()
+    public function __construct(
+        string $name,
+        string $streetAddress,
+        string $streetAddressComplementary,
+        string $postalCode,
+        string $city
+    )
     {
         $this->id = Uuid::uuid4()->toString();
+        $this->name = $name;
+        $this->streetAddress = $streetAddress;
+        $this->streetAddressComplementary = $streetAddressComplementary;
+        $this->postalCode = $postalCode;
+        $this->city = $city;
     }
 
     public function getId(): ?int
```

Comment:
Id is supposed to be a mandatory UuidInterface, isn't it?

Reply:
I have removed this, the return type is dependent on what the platform supports but by default it is converted to varchar 
https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/basic-mapping.html

Reply:
Hum indeed... Sorry about that.
Could you change the type of the id from `* @ORM\Column(type="guid")` to `* @ORM\Column(type="uuid")`, checking that [Doctrine UUID](https://github.com/ramsey/uuid-doctrine) is installed correctly.
Please do not use the generator though, as we want objects to be always in a valid state (i.e. have an id). So not:
```
* @ORM\GeneratedValue(strategy="CUSTOM")
* @ORM\CustomIdGenerator(class="Ramsey\Uuid\Doctrine\UuidGenerator")
```

Thanks again for your interest and work!